great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -91,6 +92,62 @@ class ExpectColumnValuesToBeBetween(ColumnMapExpectation):
         "allow_cross_type_comparisons",
         "mostly",
         "parse_strings_as_datetimes",
+        "auto",
+        "profiler_config",
+    )
+
+    default_profiler_config: RuleBasedProfilerConfig = RuleBasedProfilerConfig(
+        name="expect_column_values_to_be_between",  # Convention: use "expectation_type" as profiler name.
+        config_version=1.0,
+        variables={
+            "mostly": 1.0,
+            "strict_min": False,
+            "strict_max": False,
+        },
+        rules={
+            "default_column_values_between_rule": {
+                "domain_builder": {
+                    "class_name": "ColumnDomainBuilder",
+                    "module_name": "great_expectations.rule_based_profiler.domain_builder",
+                },
+                "parameter_builders": [
```

Comment:
can we use types all through this config block?

Reply:
@donaldheppner Not yet -- for now, we follow exactly the same pattern for "Base" level business objects' configuration objects as established in `DataContextConfig` and `CheckpointConfig`.  Potentially can be done in the future, but a bit too many changes if attempted to do here.